.agents/skills/debug-logs/SKILL.md

```
@@ -0,0 +1,98 @@
+### ✅ Correct
+
+```typescript
+console.log("[WS-DEBUG] subscribeSession", {
+  sessionId,
+  refCount: current + 1,
+  sentMessage: shouldSend,
+});
+```
+
+Console output:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)